bin/cli.js

```diff
@@ -23,6 +23,9 @@ console.log(`Installing dependencies for ${repoName}`);
 const installedDeps = runCommand(installDepsCommand);
 if (!installedDeps) process.exit(-1);
 
+console.log('Cleaning up...');
+runCommand(`rm -rf ${repoName}/bin`)
+
```

Comment:
```suggestion
console.log('Cleaning up...');
runCommand(`rm -rf ${repoName}/bin`)

```

Perhaps we should sanitize the variable `repoName`? (e.g. filter out stuff like `../../../`, etc.

Reply:
Created a issue on it.
Could add a check, but this is user provided input anyway, so if they want to create project into /usr/bin/, thats up to them :sweat_smile: